docs/conf.py

```diff
@@ -1 +1,2 @@
 from crate.theme.rtd.conf.crate_python import *
+html_favicon = None
```

Comment:
you should always add a newline at the end of the file :) (you can also configure your editor to do this automatically)